https://issues.redhat.com/browse/ACM-14398 (#7139)
open-cluster-management/rhacm-docs: business_continuity/backup_restore/backup_restore.adoc

[#prepare-new-hub]
== Preparing the new hub cluster 

When you need to restore backup data on a hub cluster, create a new hub cluster instead of using the primary hub cluster or a hub cluster where an earlier user created resources before the restore operation.

> In a disaster recovery scenario, when you restore the backup to the passive hub cluster, you want a hub cluster identical with the one that went down in the disaster. You do not want any other user resources, such as: applications, policies, and extra managed clusters. 

Reviewer: ```suggestion
In a disaster recovery scenario, when you restore the backup to the passive hub cluster, you want a hub cluster identical to the hub cluster that failed during the disaster. You do not want any other user resources, such as: applications, policies, and extra managed clusters. 
```

Removing directional language

> When you restore the backup on this hub cluster, if you have managed clusters managed by this hub cluster, then the restore operation might detach them and delete their cluster resources. The restore operation does this to clean up the resources that are not part of the restored backup, making the content in this hub cluster identical with the initial hub cluster. In this case, you might lose user data because the managed clusters workloads get deleted by detach action.  

Reviewer: ```suggestion
When you restore the backup on this hub cluster, if you have managed clusters managed by this hub cluster, then the restore operation might detach them and delete their cluster resources. The restore operation does this to clean up the resources that are not part of the restored backup, making the content in this hub cluster identical with the initial hub cluster. In this case, you might lose user data because the managed cluster workloads get deleted by detach action.  
```
Typo I think?
